rest/server_context.go

```diff
@@ -13,15 +13,14 @@ import (
 	"errors"
 	"fmt"
 	"net/http"
-	"net/url"
 	"os"
 	"strconv"
 	"strings"
 	"sync"
 	"sync/atomic"
 	"time"
 
-	"github.com/couchbase/go-couchbase"
+	"github.com/couchbase/gocb"
```

Comment:
As discussed, it would be preferred to encapsulate the gocb/gocbcore-specific handling in base.  I think that means having something other than an explicit gocb.Cluster reference here, and instead a wrapper struct with the subset of required operations.  I think that would improve testability, but review and advise whether that's feasible.